source/crud/read-operations/sort.txt

```diff
@@ -0,0 +1,82 @@
+============
+Sort Results
+============
+
+.. default-domain:: mongodb
+
+Use a sort to change the order in which operations return documents.
```

Comment:
Nitpick: to stay consistent with `read-operations/skip.txt`, you should consider specifying "...read operations return documents".